src/app.jsx

```diff
@@ -31,52 +31,47 @@ const App = () => {
       )}
 
       <div className="actions">
-        <button
-          type="button"
-          onClick={() => setPrintModal(!printModal)}
-        >
-          {messages.add}
-        </button>
+        <div className="actions__secondary">
```

Comment:
Why don't you do "actions primary"?
Then in css `.actions.primary` ?

Reply:
because `primary` class would define style only for actions, it couldn't be reused

Reply:
you could do a bg-primary and a primary that set bg-color and color